src/CircularArrays.jl

```diff
@@ -26,19 +27,32 @@ Alias for [`CircularArray{T,1}`](@ref).
 """
 const CircularVector{T} = CircularArray{T, 1}
 
-@inline clamp_bounds(arr::CircularArray, I::Tuple{Vararg{Int}})::AbstractArray{Int, 1} = map(dim -> mod1(I[dim], size(arr.data, dim)), eachindex(I))
+# Copied from a method Base.mod, for compatibility with Julia version < 1.3,
+# where this method is not defined
+_mod(i::Integer, r::AbstractUnitRange{<:Integer}) = mod(i-first(r), length(r)) + first(r)
```

Comment:
Is it even worth keeping backwards compat?

Reply:
I think it's worth it if it's just this one simple method. But if you're planning to require Julia 1.3 anyway, we can drop this.